src/test/java/Tests.java

```diff
@@ -0,0 +1,98 @@
+import org.junit.Ignore;
+import org.testng.annotations.Test;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import static org.testng.AssertJUnit.*;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+public class Tests {
+    @Test
+    public void TestJSonParserWriteToFile(){
+
+            JsonParser jsonParser = new JsonParser();
+            jsonParser.writeToFile(new Cart("Belekas"));
+            Path path = Paths.get("src//main//resources//Belekas.json");
+            assertTrue(Files.exists(path));
```

Comment:
in case of fail we'll see something like `Expected true but found false`, not quite informative.
make sure you left the assertion error message on each assert. example `assertTrue(String.format("File %s doesn't exist", path), Files.exists(path));`